applications/plugins/SofaCarving/CarvingManager.cpp

```diff
@@ -91,11 +87,6 @@ void CarvingManager::init()
         std::vector<core::CollisionModel*> models;
         getContext()->get<core::CollisionModel>(&models, core::objectmodel::Tag("CarvingSurface"), core::objectmodel::BaseContext::SearchRoot);
     
-        // extend the research to model without the tag. 
-        if (models.empty())
```

Comment:
No because, if there is no models, this means no collision models with the tag CarvingSurface have been found. The user need to set that tag to insure a correct behavior of the CarvingManager.
He will be informed by: 
> error: [CarvingManager(carvingManager1)] m_surfaceCollisionModels not found. Set tag 'CarvingSurface' to the right collision models.

The old behavior I removed was to pick up the first randomly collision model encountered in the graph.
```
if (models.empty())
getContext()->get<core::CollisionModel>(&models, core::objectmodel::BaseContext::SearchRoot);
```